modules/ROOT/pages/appendix/version-4.adoc

```diff
@@ -1,6 +1,6 @@
 :description: This section lists all Bolt message state transitions available in version 4.x.
 
-= Bolt message state transitions in version 4.x
+= Bolt message state transitions in version 4.x - 5.0
```

Comment:
I'm not sure this is a long term solution. I assume the table for version 4.x refers to the LTS, and that it became stable and fixed only when version 5 came about. What's gonna happen if Bolt introduces more changes in 5.5, for example? Are we going to have another whole page? 
All in all, I understand its purpose, but this `5.0` dropped close to `4.x` is quite confusing. 
I don't know how it was handled previously, but worth a discussion with the team.

Reply:
ill double check sorry i forgot my push up would notify you i wanted to run through it with the team before i got you to look again! i think a larger re-work is needed in general we are going to have. meeting to discuss this as well